docs/source/tracking.rst

```diff
@@ -83,6 +83,9 @@ and a variety of remote file storage solutions. For storing runs and artifacts,
 MLflow entities (runs, parameters, metrics, tags, notes, metadata, etc), the artifact store persists artifacts
 (files, models, images, in-memory objects, or model summary, etc).
 
+The MLflow client can be configured with an HTTP proxy, passing artifact requests through the tracking server to store and retrieve artifacts without having to specify the fully qualified path to the artifacts.
```

Comment:
This is an MLflow server configuration, not a client configuration.

Reply:
yikes yeah. fixed.